bmi_lstm.py

```diff
@@ -509,7 +509,8 @@ def set_value(self, var_name, value):
             # jmframe: this next line is basically a duplicate. 
             # I guess we should stick with the attribute names instead of a dictionary approach. 
             self._values[var_name] = value[0]
-        except TypeError:
+        # JLG 03242022: this isn't really an "error" block as standalone considers value as scalar?
+        except:
```

Comment:
It is only an "error" if the `setattr( self, var_name, value[0] )` or `self._values[var_name] = value[0]` raises a type error...this happens when you try to index `value` but `value` is not an array.  This is essentially duck typing to support the case where ngen sets value from an array.  If this isn't working for standalone cases, we may need to more carefully inspect the input type of `value`

Reply:
Guess this was my cheap attempt at NOT doing exactly this... :)
> we may need to more carefully inspect the input type of `value`

I'll look into changing how `value` is passed in `set_value()` in `run-lstm-with-bmi.py` then reopen PR. 
Thanks @hellkite500

Reply:
@hellkite500 `set_value()` now accepts singleton array for standalone script. -thx